packages/dd-trace/src/plugins/database.js

```diff
@@ -1,9 +1,63 @@
 'use strict'
 
 const StoragePlugin = require('./storage')
+const TextMapPropagator = require('../opentracing/propagation/text_map.js')
+const Config = require('../../src/config')
+const config = new Config()
+const textMap = new TextMapPropagator(config)
+const carrier = {}
 
 class DatabasePlugin extends StoragePlugin {
   static get operation () { return 'query' }
+
+  constructor (...args) {
+    super(...args)
+    this.serviceTags = {
+      dddbs: '',
+      encodedDddbs: '',
+      dde: '',
+      encodedDde: '',
+      ddps: '',
+      encodedDdps: '',
+      ddpv: '',
+      encodedDdpv: ''
+    }
+  }
+
+  createDBMPropagationCommentService () {
+    if (this.serviceTags.dddbs !== this.config.service) {
```

Comment:
This is the same code repeated 4 times with only 2 property names changing, and thus could be extracted to a function called 4 times instead of repeating the code.

Reply:
created a helper called `propagationHelper` which will take in the tags and update the local service tags object